scipy/stats/qmc.py

```diff
@@ -1156,6 +1202,37 @@ def random(self, n_samples=1):
         self.num_generated += n_samples
         return sample
 
+    def random_base2(self, m=1):
+        """Draw point(s) from the Sobol' sequence.
+
+        This function draws :math:`n=2^m` points in the parameter space
+        ensuring the balance properties of the sequence.
+
+        Parameters
+        ----------
+        m : int
+            Exponent (power of 2) to calculate the number of samples to
```

Comment:
this is somewhat confusing, could be interpreted that the exponent itself should be a power of 2

Reply:
I will change it with @ArtOwen comments.